src/doc/trpl/error-handling.md

```diff
@@ -181,6 +181,8 @@ match version {
 This function makes use of an enum, `ParseError`, to enumerate the various
 errors that can occur.
 
+The [`Debug`](../core/fmt/trait.Debug.html) trait is what let's us print the enum value using the `{:?}` format operation.
```

Comment:
I added a second commit to same branch in my fork - https://github.com/rust-lang/rust/commit/f6e5369e53f5444240aa983c6e6be6725932e399 - but I'm a bit confused on how this works - should I do a separate PR?